pdf/model/writer.go

```diff
@@ -676,17 +676,17 @@ type EncryptOptions struct {
 type EncryptionAlgorithm int
 
 const (
-	// RC4_128bit uses RC4 encryption (128 bit)
-	RC4_128bit = EncryptionAlgorithm(iota)
-	// AES_128bit uses AES encryption (128 bit, PDF 1.6)
-	AES_128bit
-	// AES_256bit uses AES encryption (256 bit, PDF 2.0)
-	AES_256bit
+	// RC4128bit uses RC4 encryption (128 bit)
+	RC4128bit = EncryptionAlgorithm(iota)
```

Comment:
RC4128bit is confusing. RC4_128bit clearly better.  Would keep the others same style for consistency here.